chapters/annotations.tex

```diff
@@ -1188,6 +1188,7 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
 \end{lstlisting}
 \end{example}
 
+A component or a short replaceable class may have the following annotation:
```

Comment:
I think at least the word _declaration_ is missing here; the class thing is a _definition_, right?

Reply:
Yes, I have now changed that.

Reply:
Looks good.  Resolving.